tutorials/gui_config.md

```diff
@@ -28,10 +51,9 @@ through the command line or the SDF file.
 
 ### Default configuration
 
-Let's try this in practice. First, let's open Gazebo without passing
-any arguments:
+Let's try this in practice. First, let's open the default Gazebo world:
 
-`gz sim`
+`gz sim default.sdf`
```

Comment:
also need to update line 82 below to be `gz sim default.sdf`

Reply:
Updated in https://github.com/gazebosim/gz-sim/pull/2497/commits/303e18ba127b39c75d6c9873d26a7d06f266e7f6